packages/meson.rb

```diff
@@ -3,24 +3,24 @@
 class Meson < Package
```

Comment:
How did meson end up in this PR?  Didn't we just merge that recently?

Reply:
Ugh. My master merges must be screwed up.

Reply:
Fixed.